modules/ocf_docker/manifests/deckschrubber.pp

```diff
@@ -0,0 +1,20 @@
+class ocf_docker::deckschrubber {
+  package { 'deckschrubber': }
+
+  file { ['/opt/docker', '/opt/docker/registry']:
+    ensure => directory,
+    mode   => '0755',
+  } ->
+  file { '/opt/docker/registry/garbage-collect':
+    source  => 'puppet:///modules/ocf_docker/garbage-collect',
+    mode    => '0755',
+    require => Package['deckschrubber'],
```

Comment:
This script also requires `File['/var/lib/registry:/var/lib/registry']` and `File['/etc/docker/registry/config.yml']` to exist.

Reply:
You're right about `/var/lib/registry`. The config file is the default one included in the container though.